salt/transactional-update/10-update-rebootmgr-options.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/sbin/transactional-update cleanup dup salt
```

Comment:
I think we should leave it at 12hrs - we do not currently have a an auto update timer, when we get that we can set the pillar value to whatever that is.

Right now, if we have a 24hr window, and a 24 hour update window, it could 48 hours before an update gets applied.

For normal updates this might be fine, but there is some updates that will need much faster deployments. (libssl and similar)